go/ql/test/library-tests/semmle/go/concepts/LoggerCall/glog.go

```diff
@@ -31,11 +31,11 @@ func glogTest() {
 	glog.Warningln(text)       // $ logger=text
 
 	// components corresponding to the format specifier "%T" are not considered vulnerable
-	glog.Errorf("%s: found type %T", text, v)   // $ logger="%s: found type %T" logger=text logger=v
-	glog.Exitf("%s: found type %T", text, v)    // $ logger="%s: found type %T" logger=text logger=v
-	glog.Fatalf("%s: found type %T", text, v)   // $ logger="%s: found type %T" logger=text logger=v
-	glog.Infof("%s: found type %T", text, v)    // $ logger="%s: found type %T" logger=text logger=v
-	glog.Warningf("%s: found type %T", text, v) // $ logger="%s: found type %T" logger=text logger=v
+	glog.Errorf("%s: found type %T", text, v)   // $ logger="%s: found type %T" logger=text
+	glog.Exitf("%s: found type %T", text, v)    // $ logger="%s: found type %T" logger=text
+	glog.Fatalf("%s: found type %T", text, v)   // $ logger="%s: found type %T" logger=text
+	glog.Infof("%s: found type %T", text, v)    // $ logger="%s: found type %T" logger=text
+	glog.Warningf("%s: found type %T", text, v) // $ logger="%s: found type %T" logger=text
```

Comment:
Is this change because the tests in the first commit were written so that they pass without the fix?

Reply:
I marked them as SPURIOUS.